Dockerfile

```diff
@@ -2,8 +2,13 @@ FROM php:8-cli-alpine
 
 WORKDIR /etc/rector
 
-# required for composer patches
-RUN apk add --no-cache patch
+RUN apk add --no-cache patch \
+    && apk add --no-cache --virtual .build-deps $PHPIZE_DEPS icu-dev \
+    && pecl install xdebug \
+    && docker-php-ext-enable xdebug \
```

Comment:
What about performance hit for normal Rector run? We had couple reports of slow Rector, then it took couple of weeks to find the default xdebug is to blame :) this cannot be here.

Reply:
By default XDebug is transparent because of `XDEBUG_MODE` fallback value `off`. It must be enabled explicitly by setting `debug` in `.env` file. Extension enabled was a problem prior to v3, now it can be always enabled and there's no overhead.

Reply:
Coudl you link this knowledge in these lines? We're not using Docker ready, so we are aware of this explicitly

Reply:
It's not Docker-related knowledge, it's how XDebug v3 works 😉 But I've added some info to `CONTRIBUTING.md`.